ansys/dpf/core/settings.py

```diff
@@ -0,0 +1,35 @@
+from ansys.dpf.core.misc import module_exists
+from ansys.dpf.core import misc
+
+def disable_off_screen_rendering() -> None:
+    # enable matplotlib off_screen plotting to avoid test interruption
+    if module_exists("matplotlib"):
+        import matplotlib as mpl
+
+        mpl.use("Agg")
+
+    # enable off_screen plotting to avoid test interruption
+    if module_exists("pyvista"):
+        import pyvista as pv
+
+        pv.OFF_SCREEN = True
+
+def set_default_pyvista_config():
+    # Configure PyVista's ``rcParams`` for dpf
+    if module_exists("pyvista"):
+        import pyvista as pv
+
+        pv.rcParams["interactive"] = True
```

Comment:
Consider using the new pyvista theme API.  See:

https://docs.pyvista.org/api/plotting/_autosummary/pyvista.Plotter.theme.html?highlight=theme#pyvista.Plotter.theme

and

https://docs.pyvista.org/user-guide/themes.html?highlight=theme

and

https://docs.pyvista.org/examples/02-plot/themes.html?highlight=theme

Requires pyvista >= 0.31.0